conda/testing/__init__.py

```diff
@@ -185,20 +185,20 @@ def __call__(self, *argv: str) -> tuple[str, str, int]:
 
         # mock legacy subcommands
         if argv[0] == "env":
-            from conda_env.cli.main import create_parser, do_call
+            from conda.cli.main_env import configure_parser, execute
 
             argv = argv[1:]
 
             # parse arguments
-            parser = create_parser()
+            parser = configure_parser()
             args = parser.parse_args(argv)
 
             # initialize context and loggers
             context.__init__(argparse_args=args)
             init_loggers()
 
             # run command
-            code = do_call(args, parser)
+            code = execute(args, parser)
 
```

Comment:
Assuming we have added `conda.cli.main_env.configure_parser` to `conda.cli.conda_argparse.generate_parser` this entire legacy clause for `conda env` is unecessary